tsl/test/expected/dist_ddl.out

```diff
@@ -2533,6 +2533,246 @@ NOTICE:  adding not-null constraint to column "time"
 INSERT INTO hyper SELECT t, ceil((random() * 5))::int, random() * 80
 FROM generate_series('2019-01-01'::timestamptz, '2019-01-05'::timestamptz, '1 minute') as t;
 ANALYZE hyper;
+--
+-- Test hypertable distributed defaults
+--
+SHOW timescaledb.hypertable_distributed_default;
+ timescaledb.hypertable_distributed_default 
+--------------------------------------------
+ auto
+(1 row)
+
+SHOW timescaledb.hypertable_replication_factor_default;
+ timescaledb.hypertable_replication_factor_default 
+---------------------------------------------------
+ 1
+(1 row)
+
+/* CASE1: create_hypertable(distributed, replication_factor) */
+-- defaults are not applied
+CREATE TABLE drf_test(time TIMESTAMPTZ NOT NULL);
+\set ON_ERROR_STOP 0
+SELECT create_hypertable('drf_test', 'time', distributed=>false, replication_factor=>1);
+ERROR:  operation not supported
+\set ON_ERROR_STOP 1
+SELECT create_hypertable('drf_test', 'time', distributed=>true, replication_factor=>1);
+   create_hypertable    
+------------------------
+ (21,public,drf_test,t)
+(1 row)
+
+SELECT h.table_name, h.replication_factor FROM _timescaledb_catalog.hypertable h WHERE h.table_name = 'drf_test';
+ table_name | replication_factor 
+------------+--------------------
+ drf_test   |                  1
+(1 row)
+
+DROP TABLE drf_test;
+/* CASE2: create_hypertable(distributed) */
+-- auto
+SET timescaledb.hypertable_distributed_default TO 'auto';
+-- create regular hypertable by default
+CREATE TABLE drf_test(time TIMESTAMPTZ NOT NULL);
+SELECT create_hypertable('drf_test', 'time', distributed=>false);
+   create_hypertable    
+------------------------
+ (22,public,drf_test,t)
+(1 row)
+
+SELECT h.table_name, h.replication_factor FROM _timescaledb_catalog.hypertable h WHERE h.table_name = 'drf_test';
```

Comment:
Maybe use the `timescaledb_information.hypertables` view for this where there's a "distributed" bool column.